backend/schema.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE tokens(
+  id serial PRIMARY KEY,
+  token_value varchar(255)
+);
+
+CREATE TABLE baskets(
+  id serial PRIMARY KEY,
+  name varchar(255) UNIQUE NOT NULL,
+  token_id bigint NOT NULL REFERENCES tokens(id) ON DELETE CASCADE
+);
+
+CREATE TABLE notifications(
+  id serial PRIMARY KEY,
+  basket_id bigint NOT NULL REFERENCES baskets(id) ON DELETE CASCADE,
+  sent_at timestamp NOT NULL,
+  method varchar(6) NOT NULL,
```

Comment:
Apparently there are other HTTP methods like `CONNECT` and `OPTIONS`. Should we increase it to a safer number so we avoid having issues with these edge cases?

How about `varchar(16)` or whatever you feel is safe :)